src/librbd/Types.h

```diff
@@ -114,6 +118,29 @@ struct SnapInfo {
   }
 };
 
+enum {
+  OPEN_FLAG_SKIP_OPEN_PARENT = 1 << 0,
+  OPEN_FLAG_OLD_FORMAT = 1 << 1,
+  OPEN_FLAG_IGNORE_MIGRATING = 1 << 2,
+};
+
+struct WatcherInfo {
+  std::string addr;
+  int64_t watcher_id = 0;
+  uint64_t cookie = 0;
+  uint32_t timeout_seconds = 0;
+  bool mirroring = false;
+  bool me = false;
```

Comment:
Nit: too non-descriptive (e.g. lock owner)

Reply:
me is not necessary the lock owner. May be "own" or "owner"?

Reply:
Perhaps ```local_instance``` / ```my_instance```?